src/include/daos/agent.h

```diff
@@ -26,15 +26,30 @@
 
 #include <daos/drpc.h>
 
+int dc_agent_init(void);
+void dc_agent_fini(void);
+
+extern char *dc_agent_sockpath;
+
 /**
- * Default Unix Domain Socket path for the DAOS agent dRPC connection
+ * Default runtime directory for daos_agent
  */
-#define DEFAULT_DAOS_AGENT_DRPC_SOCK "/var/run/daos_agent/agent.sock"
-
+#define DAOS_AGENT_DRPC_DIR "/var/run/daos_agent/"
 /**
  * Environment variable for specifying an alternate dRPC socket path
  */
-#define DAOS_AGENT_DRPC_SOCK_ENV "DAOS_AGENT_DRPC_SOCK"
+#define DAOS_AGENT_DRPC_DIR_ENV "DAOS_AGENT_DRPC_DIR"
+
+/**
+ * Socket name used to craft path from environment variable
+ */
+#define DAOS_AGENT_DRPC_SOCK_NAME "agent.sock"
+
+/**
+ * Default Unix Domain Socket path for the DAOS agent dRPC connection
+ */
+#define DEFAULT_DAOS_AGENT_DRPC_SOCK 	DAOS_AGENT_DRPC_DIR \
```

Comment:
(style)  please, no space before tabs

(style)  Macros with complex values should be enclosed in parentheses

Reply:
fixed